ui/components/multichain/asset-picker-amount/asset-picker/asset-picker.tsx

```diff
@@ -199,7 +199,6 @@ export function AssetPicker({
                 borderRadius={isNFT ? BorderRadius.LG : BorderRadius.full}
                 src={primaryTokenImage}
                 size={AvatarTokenSize.Md}
-                showHalo={!isNFT}
                 name={symbol}
                 {...(isNFT && { backgroundColor: BackgroundColor.transparent })}
               />
```

Comment:
It would be great if this component had a story, @bergeron, so it can be viewed in isolation for UI updates like this. I had to search for where it was used and found it in `AssetPickerAmount`, so I checked that story but it increased review time.